build.rs

```diff
@@ -11,7 +11,12 @@ fn sdk_path(target: &str) -> Result<String, std::io::Error> {
 
     let sdk = if target.contains("apple-darwin") {
         "macosx"
-    } else if target.contains("apple-ios") {
+    } else if target == "x86_64-apple-ios" || target == "i386-apple-ios" {
+        "iphonesimulator"
+    } else if target == "aarch64-apple-ios"
+        || target == "armv7-apple-ios"
```

Comment:
I'm on the fence for including them at all as potential targets at all as armv7 and down have been set demoted to [3rd tier.](https://blog.rust-lang.org/2020/01/03/reducing-support-for-32-bit-apple-targets.html). It looks like [armv7 was on at newest the iPhone 5c.](https://medium.com/@edisonlo/ios-framework-file-was-built-for-x86-64-which-is-not-the-architecture-being-linked-arm64-linker-4d3780669aa3).